### ISSUE TITLE
[16.0] base: edit internal note in chatter

### ISSUE THREAD

**Issue by clementelabore** [odoo/odoo#183936]
### Odoo Version

- [x] 16.0
- [X] 17.0
- [X] 18.0
- [ ] Other (specify)

### Steps to Reproduce

1. Click on "Log note" in the chatter on any record
2. Click on "Full composer" button (bottom right of textarea)
3. Add a note with rich text (Bulleted list for example)
4. Click on "Log" button to save your note
5. Click on the pencil icon to edit your note

=> your text formatting has disappeared

**Comment by seb-odoo** [comment]
Hello, thank you for the report.

This is a known issue that is hard to resolve due to the small composer being simple text, not rich text.

We are making ongoing effort to resolve it, the current solution is going towards using rich editor also when using the small composer or when editing, but it takes a lot of effort to properly to do it.

It will not be fixed in existing version, but we want a long term solution so I'll close the issue as it's already on our map.

**Comment by stephansainleger** [comment]
Hi @seb-odoo ! 
Thanks for your answer. I understand this issue won't be solved in version 18, is that right? Do you know, and can you share with us, the roadmap for this correction. Can we expect it for version 19.0 for instance?
Thanks a lot for your help! :-)

**Comment by seb-odoo** [comment]
Indeed, this is not resolved in Odoo 18, but we after many unsuccessful attempts, we hope to merge https://github.com/odoo/odoo/pull/188503 soon. If we succeed with this version of the change then yes it will be in Odoo 19, but this is not a promise or guaranteed.
